setup.py

```diff
@@ -0,0 +1,46 @@
+from setuptools import setup, find_packages
+from manheim_cloudmapper.version import VERSION, PROJECT_URL
+
+with open('README.md') as f:
+    long_description = f.read()
+
+requires = [
+    'boto3',
+    'docutils>=0.10,<0.15',
+    'tabulate>=0.8.0,<0.9.0',
+    # for building generated policy docs
+    'sphinx>=1.8.0,<1.9.0',
+    'sphinx_rtd_theme',
+    'pandas',
+    'premailer'
+]
+
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Information Technology',
+    'Intended Audience :: System Administrators',
+    'License :: OSI Approved :: Apache Software License',
+    'Natural Language :: English',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python :: 3',
+    'Topic :: System :: Distributed Computing',
+    'Topic :: System :: Systems Administration',
+    'Topic :: Utilities'
+]
+
+setup(
```

Comment:
Since this isn't an actual python package that can be cleanly installed, let's remove the classifiers list (lines 18-31) and then drop the ``setup()`` call to only the [absolute minimum arguments](https://setuptools.readthedocs.io/en/latest/setuptools.html#basic-use), ``name``, ``version``, and ``packages``.